### DISCUSSION TITLE
Vespertine l.l.c.

### DISCUSSION THREAD

**Discussion by richantz503** [dotnet/maui#8962] [Ideas]
I have a business idea that I've been building on for several years now, and has had very positive reactions from several executives and investors that could possibly be the next big thing. Only I can't build the program I know nothing about coding! I need a savvy, creative, brilliant, programmer to help me develop this dream.

**Comment by xtuzy** [comment]
You should go to the recruitment website

**Reply by richantz503** [reply]
Where's that at?

**Reply by hartez** [reply]
https://www.linkedin.com/
